templates/corporate/billing/billing.html

```diff
@@ -571,4 +588,41 @@ <h1 class="modal__title dialog_heading">
         </div>
     </div>
 </div>
+<div id="confirm-toggle-license-management-modal" class="micromodal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1">
+        <div class="modal__container" role="dialog" aria-modal="true" aria-labelledby="dialog_title">
+            <header class="modal__header">
+                <h1 class="modal__title dialog_heading">
+                    Confirm switching to
+                    {% if automanage_licenses %}
+                    manual
+                    {% else %}
+                    automatic
+                    {% endif %}
+                    license management
+                </h1>
+                <button class="modal__close" aria-label="{{ _('Close modal') }}" data-micromodal-close></button>
+            </header>
+            <main class="modal__content">
+                <p>
+                    Are you sure you want to switch to
+                    <a href="https://zulip.com/help/self-hosted-billing#how-does-automatic-license-management-work">
```

Comment:
I think these could be relative links? we have several similar links in this page that use them...

Reply:
Opened #31772